examples/minesweeper/README.md

```diff
@@ -18,7 +18,9 @@ Choose a run configuration for an appropriate target in Android Studio and run i
 
 `./gradlew desktopApp:run`
 
-## Run JS in browser with WebAssembly Skia via Gradle
+## Run JS in browser
```

Comment:
Let's remove this section `Run JS in browser`. See the comment to the `chat` example.

Reply:
Done